### PR TITLE
14965 Fixed register notation disabled for SP/GP filling

### PR DESCRIPTION
**Pull request by JazzarKarim**
Issue #: /bcgov/entity#14965

Description of changes:

Fixed the ability to file register notation for sp/gp filling

Screenshot (17)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

**Comment by severinbeauvais** [comment]
@JazzarKarim Is this PR still valid?

**Comment by JazzarKarim** [comment]
@severinbeauvais Yes I just checked.

**Comment by JazzarKarim** [comment]
> 

I cannot assign myself or add reviewers. Maybe I don't have the required access?

**Review by severinbeauvais — APPROVED** [review]
Looks good but please update the app version and commit the 2 package files.

**Comment by severinbeauvais** [comment]
Also please wait for some other reviews before merging. I do not want to be the primary or sole reviewer :)

**Review by ChrisGabel23 — APPROVED** [review]
This has to be the record for smallest code change - one character added! 😎

**Comment by JazzarKarim** [comment]
> This has to be the record for smallest code change - one character added! 😎

It's now a two character change if you include the app version change :smirk:

**Pull request merged**
